README.md

```diff
@@ -364,7 +364,23 @@ The `settingsOptions` parameter is explained in the next section.
 
 **Note:** This method throws a `CreateJobError` if job creation fails (e.g., 404 error).
 
-3. **Monitor job progress** - Track the status of your job:
+3. **Attach to a code generation job** - Use the hook to attach to existing jobs:
```

Comment:
I think this instruction is confusing because it appears that it's a mandatory 3rd step for getting started. But that's not the case.

You can mention it as a new note below, or in a section.